tools/ci_build/github/android/build_aar_package.py

```diff
@@ -73,12 +75,15 @@ def _parse_build_settings(args):
 
 def _build_aar(args):
     build_settings = _parse_build_settings(args)
-    build_dir = args.build_dir
+    build_dir = os.path.abspath(args.build_dir)
 
     # Setup temp environment for building
-    my_env = os.environ.copy()
-    my_env['ANDROID_HOME'] = build_settings['android_sdk_path']
-    my_env['ANDROID_NDK_HOME'] = build_settings['android_ndk_path']
+    _env = os.environ.copy()
+    _env['ANDROID_HOME'] = os.path.abspath(args.android_sdk_path)
+    _env['ANDROID_NDK_HOME'] = os.path.abspath(args.android_ndk_path)
+
+    # If not using shell on Window, will not be able to find commands in path
```

Comment:
>will not be able to find commands in path [](start = 36, length = 41)

just curious, which command requires shell=True? i guess build.py doesn't - if so maybe consider `sys.executable` instead of `python3`?

Reply:
Actually it was Gradle, which cannot be found on Windows unless we use shell=True,